graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -82,25 +93,26 @@ def rename_schema(ast, renamings):
     )
 
 
-def _rename_types(ast, renamings, query_type, scalars):
+def _rename_types(
+    ast: DocumentNode, renamings: Dict[str, str], query_type: str, scalars: Set[str]
+) -> Tuple[DocumentNode, Dict[str, str]]:
     """Rename types, enums, interfaces using renamings.
 
     The query type will not be renamed. Scalar types, field names, enum values will not be renamed.
 
     The input AST will not be modified.
 
     Args:
-        ast: Document, the schema that we're returning a modified version of
-        renamings: Dict[str, str], mapping original type/interface/enum name to renamed name. If
-                   a name does not appear in the dict, it will be unchanged
-        query_type: str, name of the query type, e.g. 'RootSchemaQuery'
-        scalars: Set[str], the set of all scalars used in the schema, including user defined
-                 scalars and and used builtin scalars, excluding unused builtins
+        ast: The schema that we're returning a modified version of
```

Comment:
Interesting, I forgot this is a schema AST — can we rename it to `schema_ast` while we're at it?